bin/compile

```diff
@@ -35,16 +38,38 @@ cp -n $BP_DIR/scripts/Procfile $BUILD_DIR/Procfile
 echo "-----> Copying static.json to application slug"
 cp -n $BP_DIR/scripts/static.json $BUILD_DIR/static.json
 
-echo "-----> Installing Bower package manager"
 cd $BUILD_DIR
-npm install bower
+if [ -d "./bower_components" ]; then
```

Comment:
If you go down this path, you'll also want to be able to clear the bower cache when `bower.json` changes since it doesn't have a lockfile. I do that in the [ember-cli-deploy buildpack](https://github.com/heroku/heroku-buildpack-ember-cli-deploy/blob/master/buildpack/mrblib/buildpack/commands/compile/bower.rb#L24-L29).